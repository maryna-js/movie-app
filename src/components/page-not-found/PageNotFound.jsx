import React from 'react';
import { Link } from 'react-router-dom';

export const PageNotFound = () => (
  <div className="bg-gray-200 flex w-full h-screen flex-col justify-center items-center">
    <div className="font-normal text-3xl text-blue-900">Page Not Found</div>
    <div className="text-6xl text-blue-900 drop-shadow my-4">404</div>
    <Link to='/'>
      <button className="text-gray-800 text-uppercase text-base">
        GO BACK TO HOME
      </button>
    </Link>
  </div>
);
