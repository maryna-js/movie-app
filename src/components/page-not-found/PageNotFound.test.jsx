import React from 'react';
import { render } from '@testing-library/react';
import { createMemoryHistory } from 'history';
import { BrowserRouter as Router } from 'react-router-dom';
import { PageNotFound } from './PageNotFound';

describe('PageNotFound', () => {
    const history = createMemoryHistory();
    const getComponent = () =>
        render(
            <Router navigator={history}>
                <PageNotFound />
            </Router>
        );
    it('Should render correctly', () => {
        expect(getComponent().asFragment()).toMatchSnapshot();
    });
});