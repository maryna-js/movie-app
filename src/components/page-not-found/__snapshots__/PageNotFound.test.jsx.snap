// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-gray-200 flex w-full h-screen flex-col justify-center items-center"
  >
    <div
      class="font-normal text-3xl text-blue-900"
    >
      Page Not Found
    </div>
    <div
      class="text-6xl text-blue-900 drop-shadow my-4"
    >
      404
    </div>
    <a
      href="/"
    >
      <button
        class="text-gray-800 text-uppercase text-base"
      >
        GO BACK TO HOME
      </button>
    </a>
  </div>
</DocumentFragment>
`;
